logstash-input-file.gemspec

```diff
@@ -20,11 +20,12 @@ Gem::Specification.new do |s|
   s.metadata = { "logstash_plugin" => "true", "logstash_group" => "input" }
 
   # Gem dependencies
-  s.add_runtime_dependency "logstash-core", ">= 2.0.0.beta2", "< 3.0.0"
+  # s.add_runtime_dependency "logstash-core", ">= 2.0.0.beta2", "< 3.0.0"
 
   s.add_runtime_dependency 'logstash-codec-plain'
   s.add_runtime_dependency 'addressable'
   s.add_runtime_dependency 'filewatch', ['>= 0.6.5', '~> 0.6']
+  s.add_dependency 'logstash-codec-multiline'
```

Comment:
it should be a runtime dependency and it should require the minimal version of the multiline that support the identity stream.